src/components/ViewLecture.js

```diff
@@ -414,15 +414,12 @@ const ViewLecture = ({ match }) => {
     return (
       <div>
         <div id='videoContainer' className='videoContainer'>
-          <ReactPlayer
-            // ref={(player) => {
-            //   this.player = player;
-            // }}
-            width='100vw'
-            // height='100vh'
-            url={`${hostUrl}/video/lecture-${url}.mp4`}
-            playing={true}
-            controls={false}
+
+          <ReactWebMediaPlayer
```

Comment:
This looks fine but Please check if this library has enough weekly downloads in lakhs atleast?